enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -866,36 +867,36 @@ func (l *FileCacheLoader) cacheCOW(ctx context.Context, name string, remoteInsta
 		},
 	}
 
+	emptyData := make([]byte, cow.ChunkSizeBytes())
+	allZerosDigest, err := digest.Compute(bytes.NewReader(emptyData), repb.DigestFunction_BLAKE3)
```

Comment:
there is a function in digest.go you could probably use for this:

func IsEmptyHash(d *repb.Digest, digestFunction repb.DigestFunction_Value) bool {}

Reply:
Saw that but I checked that it generates a different hash for empty data vs data filled with 0s